apollo-router/src/context.rs

```diff
@@ -80,6 +80,18 @@ impl Context {
         }
     }
 
+    /// Insert a value int the context using the provided key and value.
+    ///
+    /// Semantics:
+    ///  - If the operation fails, then the pair has not been inserted.
```

Comment:
I don't see any Result in the function definition ? Oo

Reply:
Right, I should remove that comment. The insert method can fail because it serializeq and deserializes json values, but this one does not